packages/@sparrow-workspaces/src/features/rest-explorer/components/response-body/ResponseBody.svelte

```diff
@@ -32,12 +33,17 @@
       : ''}"
   >
     {#if apiState.bodyLanguage === RequestDataTypeEnum.IMAGE}
-      <!-- 
-        --
-        -- Reponse content-type set to image,
-        -- 
-      -->
-      <SparrowLogo />
+      {#if !imageHasError}
+        <!-- render image from base64 -->
+        <img
+          src={`data:image/jpeg;base64,${response.body}`}
+          alt="Sparrow logo"
+          on:error={() => (imageHasError = true)}
```

Comment:
will this work for PNG also?

Reply:
Yes and for Stability we can set it from content type i will commit this changes

Reply:
working for every type of image response :-
jpeg : https://picsum.photos/300/200
png: https://dummyimage.com/300x200.png 
GIF : https://cataas.com/cat/gif
webp: https://picsum.photos/300/200.webp